HISTORY.rst

```diff
@@ -0,0 +1,15 @@
+.. :changelog:
+
+
+Release History
+---------------
+
+
+0.21.2 (2016-04-18)
++++++++++++++++++++
+
+
+* Switched from `django.utils.importlib` to Python's standard `importlib` to
```

Comment:
Also, this release number has already been tagged: https://github.com/mobify/dj-saml-idp/releases/tag/0.21.2